### PR TITLE
RF arguments for cite:  kind -> tags, level -> path, use -> desc

### LINKED ISSUE #8: RFC: either rename "kind" and "level" into some thing more descriptive
Hi @mvdoc , we should decide either it would stay as is:

- **level** is just a string with some attached semantic on the prefix of it, then we should document/check on given values, or should it may be become differently defined, 
- **kind**  I have recently added to state what kind of reference it is -- canonical reference to the method/implementation, or example use, or educational tutorial

Should we stick to those names or may be there could be better one chosen?
May be "kind"  could simply be refactored into "tags" list, so multiple tags could be provided and we would just have some "semantic" associated with "canonical",  "use", "edu", and the rest might come useful later... 

What do you think?

Anyone else reading -- your input would be valuable!

### PR DESCRIPTION
**Pull request by yarikoptic**
Aims to Close #8 

May be that was too much with "desc" but "use" also sounded a bit inappropriate

### CODE REVIEW

File: duecredit/collector.py [hunk 1]
```diff
@@ -67,16 +72,44 @@ def __repr__(self):
...
+    @property
+    def module(self):
+        if not self.path:
+            return None
+        return self.path.split(':', 1)[0]
+
+    def __contains__(self, entry):
+        """Checks if provided entry 'contained' in this one given its path
+
+        If current entry is associated with a module, contained will be and entry
```
Comment (mvdoc) [inline duecredit/collector.py]: typo: an entry

**Comment by mvdoc** [comment]
I think you are right that it's better to expand the keywords and tags (`desc`, `impl`, etc)

**Comment by yarikoptic** [comment]
@mvdoc so something like now? ;)

File: duecredit/collector.py (hunk 1 above)
Comment (yarikoptic) [inline duecredit/collector.py]: pushed a fix, thanks

**Comment by mvdoc** [comment]
:+1:

**Comment by yarikoptic** [comment]
Btw @mvdoc, May be doc instead of description? Much shorter

**Comment by mvdoc** [comment]
shorter but ambiguous? I like description :-)

**Pull request merged**